### ISSUE TITLE
What is num_per_class variable s3dis.py

### ISSUE THREAD

**Issue by Husnain-Javid** [guochengqian/PointNeXt#129]
#Hi @guochengqian, thanks for your work. I have a quick question.

What is num_per_class variable in openpoints/dataset/s3dis/s3dis.py

num_per_class = np.array([3370714, 2856755, 4919229, 318158, 375640, 478001, 974733,
                              650464, 791496, 88727, 1284130, 229758, 2272837], dtype=np.int32)

I am trying to train PointNext on the custom dataset. I have prepared my dataset according to your provided S3DIS dataset. Which has 

```
- data
        - S3DIS
                  - s3disfull
                                - raw
                                        - .....npy
                                - processed
                                        - .....pkl
```

Do I need to use s3dis.py in openponits?

**Comment by guochengqian** [comment]
It is the number of points per class (class mean the ground truth annonation: door, chair, xxx). It is only used in weighted loss function. You can follow s3dis.py to create your data loader.
